docs/1.getting-started/6.data-fetching.md

```diff
@@ -264,6 +264,67 @@ const { data, error, refresh } = await useFetch('/api/users', {
 })
 ```
 
+Note that **watching a reactive element won't change the URL fetched**. For example, this will keep fetching the same initial ID of the user because the URL is constructed at the moment the function is invoked.
```

Comment:
```suggestion
Note that **watching a reactive value won't change the URL fetched**. For example, this will keep fetching the same initial ID of the user because the URL is constructed at the moment the function is invoked.
```